Uber jar missing xml related dependencies.

**Description:**
Build and run the uber jar for the following code. 

```ballerina
import ballerina/io;
public function main() {
    xml bookXML = xml `<book>
                <name>Sherlock Holmes</name>
                <author>
                    <fname title="Sir">Arthur</fname>
                    <mname>Conan</mname>
                    <lname>Doyle</lname>
                </author>
                <!--Price: $10-->
                </book>`;
    

    io:println(bookXML.author.fname);
    io:println(bookXML["author"]["fname"]);

    io:println(bookXML.ISBN.code);
    io:println(bookXML["ISBN"]["code"]);

    io:println(bookXML.author.fname.getTextValue());
    io:println(bookXML["author"]["fname"].getTextValue());

    io:println(bookXML.author.fname@["title"]);
    io:println(bookXML["author"]["fname"]@["title"]);
}
```

Getting following error. 
```
[2019-08-26 16:43:31,314] ERROR {org.ballerinalang.jvm.scheduling.Scheduler} - Strand died 
java.lang.NoClassDefFoundError: org/apache/axiom/om/DeferredParsingException
	at hasitha.hello.a.main(a.bal:3)
	at hasitha.hello.___init.$lambda$main$(hello)
	at org.ballerinalang.jvm.scheduling.SchedulerItem.execute(Scheduler.java:453)
	at org.ballerinalang.jvm.scheduling.Scheduler.run(Scheduler.java:231)
	at org.ballerinalang.jvm.scheduling.Scheduler.runSafely(Scheduler.java:201)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.apache.axiom.om.DeferredParsingException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 6 more
 
[2019-08-26 16:43:31,328] ERROR {org.ballerinalang.jvm.util.RuntimeUtils} - org/apache/axiom/om/DeferredParsingException 
java.lang.NoClassDefFoundError: org/apache/axiom/om/DeferredParsingException
	at hasitha.hello.a.main(a.bal:3)
	at hasitha.hello.___init.$lambda$main$(hello)
	at org.ballerinalang.jvm.scheduling.SchedulerItem.execute(Scheduler.java:453)
	at org.ballerinalang.jvm.scheduling.Scheduler.run(Scheduler.java:231)
	at org.ballerinalang.jvm.scheduling.Scheduler.runSafely(Scheduler.java:201)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.apache.axiom.om.DeferredParsingException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 6 more
 ```


**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**
Ballerina 1.0.0-alpha3
Language specification 2019R2
Ballerina tool 1.0.0

openjdk version "1.8.0_202"
OpenJDK Runtime Environment Corretto-8.202.08.2 (build 1.8.0_202-b08)
OpenJDK 64-Bit Server VM Corretto-8.202.08.2 (build 25.202-b08, mixed mode)

--- a/bvm/ballerina-rt/build.gradle
+++ b/bvm/ballerina-rt/build.gradle
@@ -84,13 +84,15 @@ dependencies {
     dist 'org.apache.ws.commons.axiom:axiom-c14n:1.2.22'
     dist 'org.apache.ws.commons.axiom:axiom-dom:1.2.22'
     dist 'org.apache.ws.commons.axiom:axiom-impl:1.2.22'
+    dist 'org.apache.ws.commons.axiom:axiom-api:1.2.20'
+    dist 'commons-logging:commons-logging:1.1.1'
     dist 'io.opentracing:opentracing-api:0.31.0'
     dist 'io.opentracing:opentracing-util:0.31.0'
     dist 'org.awaitility:awaitility:3.1.6'
-    
+    dist 'com.h2database:h2:1.4.199'
+
     // Temporary adding . Need to Remove once fix #17878
     dist 'com.google.code.gson:gson:2.7'
-    dist 'com.h2database:h2:1.4.199'
 }
 
 jar {

--- a/distribution/zip/jballerina/build.gradle
+++ b/distribution/zip/jballerina/build.gradle
@@ -43,7 +43,6 @@ configurations {
 }
 
 dependencies {
-    dist 'com.h2database:h2:1.4.199'
     dist 'com.squareup.okhttp3:okhttp:3.9.1'
     dist 'com.squareup.okio:okio:1.13.0'
     dist 'com.uber.jaeger:jaeger-core:0.24.0'
@@ -66,7 +65,6 @@ dependencies {
     dist 'info.picocli:picocli:4.0.1'
     dist 'org.apache.kafka:kafka-clients:2.0.0'
     dist 'org.apache.kafka:kafka_2.11:2.0.0'
-    dist 'org.apache.ws.commons.axiom:axiom-api:1.2.20'
     dist 'io.ballerina.messaging:broker-auth:0.970.0'
     dist 'io.ballerina.messaging:broker-common:0.970.0'
     dist 'io.ballerina.messaging:broker-coordination:0.970.0'
@@ -88,7 +86,6 @@ dependencies {
     dist 'io.netty:netty-resolver:4.1.34.Final'
     dist 'io.netty:netty-tcnative-boringssl-static:2.0.23.Final'
     dist 'io.netty:netty-transport:4.1.34.Final'
-    dist 'commons-logging:commons-logging:1.1.1'
     dist 'commons-pool.wso2:commons-pool:1.5.6.wso2v1'
     dist 'org.wso2.carbon.messaging:org.wso2.carbon.messaging:2.3.7'
     dist 'org.wso2.carbon.metrics:org.wso2.carbon.metrics.core:2.3.7'

--- a/stdlib/cache/src/main/ballerina/Ballerina.toml
+++ b/stdlib/cache/src/main/ballerina/Ballerina.toml
@@ -2,12 +2,3 @@
 org-name = "ballerina"
 version = "0.0.0"
 
-[platform]
-target = "java"
-
-    [[platform.libraries]]
-    artifactId = "commons-logging"
-    version = "1.1.1"
-    path = "./lib/commons-logging-1.1.1.jar"
-    groupId = "commons-logging"
-    modules = ["cache"]

--- a/stdlib/mime/src/main/ballerina/Ballerina.toml
+++ b/stdlib/mime/src/main/ballerina/Ballerina.toml
@@ -27,20 +27,6 @@ target = "java"
     modules = ["mime"]
 
     [[platform.libraries]]
-    artifactId = "axiom-api"
-    version = "1.2.20"
-    path = "./lib/axiom-api-1.2.20.jar"
-    groupId = "org.apache.ws.commons.axiom"
-    modules = ["mime"]
-
-    [[platform.libraries]]
-    artifactId = "commons-logging"
-    version = "1.1.1"
-    path = "./lib/commons-logging-1.1.1.jar"
-    groupId = "commons-logging"
-    modules = ["mime"]
-
-    [[platform.libraries]]
     artifactId = "netty-common"
     version = "4.1.34.Final"
     path = "./lib/netty-common-4.1.34.Final.jar"

--- a/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BalRunFunctionNegativeTestCase.java
+++ b/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BalRunFunctionNegativeTestCase.java
@@ -44,9 +44,11 @@ public class BalRunFunctionNegativeTestCase extends BaseTest {
     public void testInsufficientArguments() throws BallerinaTestException {
         LogLeecher errLogLeecher = new LogLeecher("ballerina: insufficient arguments to call the 'main' function",
                                                   LeecherType.ERROR);
+        String balFile = (new File("src" + File.separator + "test" + File.separator + "resources" + File.separator +
+                                           "run" + File.separator + "file" + File.separator +
+                                           "test_main_with_multiple_typed_params" + ".bal")).getAbsolutePath();
         BMainInstance bMainInstance = new BMainInstance(balServer);
-        bMainInstance.runMain((new File("src/test/resources/run/file/test_main_with_multiple_typed_params.bal"))
-                                      .getAbsolutePath(), new LogLeecher[] { errLogLeecher });
+        bMainInstance.runMain(balFile, new LogLeecher[] { errLogLeecher });
         errLogLeecher.waitForText(LOG_LEECHER_TIMEOUT);
     }
 
@@ -54,10 +56,11 @@ public class BalRunFunctionNegativeTestCase extends BaseTest {
     public void testTooManyArguments() throws BallerinaTestException {
         LogLeecher errLogLeecher = new LogLeecher("ballerina: too many arguments to call the 'main' function",
                                                   LeecherType.ERROR);
+        String balFile = (new File("src" + File.separator + "test" + File.separator + "resources" + File.separator +
+                                           "run" + File.separator + "file" + File.separator +
+                                           "test_main_with_no_params.bal")).getAbsolutePath();
         BMainInstance bMainInstance = new BMainInstance(balServer);
-        bMainInstance.runMain((new File("src/test/resources/run/file/test_main_with_no_params.bal"))
-                                      .getAbsolutePath(), new String[] {}, new String[] { "extra" },
-                              new LogLeecher[] { errLogLeecher });
+        bMainInstance.runMain(balFile, new String[] {}, new String[] { "extra" }, new LogLeecher[] { errLogLeecher });
         errLogLeecher.waitForText(LOG_LEECHER_TIMEOUT);
     }
 }

--- a/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BalRunFunctionPositiveTestCase.java
+++ b/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BalRunFunctionPositiveTestCase.java
@@ -36,22 +36,24 @@ public class BalRunFunctionPositiveTestCase extends BaseTest {
     @Test
     public void testNoArg() throws BallerinaTestException {
         BMainInstance bMainInstance = new BMainInstance(balServer);
-        String output = bMainInstance.runMainAndReadStdOut("run", new String[]{ "test_main_with_no_params.bal" },
-                                                       (new File("src/test/resources/run/file"))
-                                                               .getAbsolutePath());
+        String output = bMainInstance.runMainAndReadStdOut("run", new String[] { "test_main_with_no_params.bal" },
+                                                           (new File("src" + File.separator + "test" + File.separator
+                                                                             + "resources" + File.separator + "run" +
+                                                                             File.separator + "file"))
+                                                                   .getAbsolutePath());
         Assert.assertEquals(output, "1");
     }
 
     @Test
     public void testMultipleParam() throws BallerinaTestException {
         BMainInstance bMainInstance = new BMainInstance(balServer);
-        String output = bMainInstance.runMainAndReadStdOut("run",
-                                                       new String[]{ "test_main_with_multiple_typed_params.bal",
-                                                               "1000", "1.0", "Hello Ballerina", "255", "true",
-                                                               "{ \"name\": \"Maryam\" }", "<book>Harry Potter</book>",
-                                                               "{ \"name\": \"Em\" }", "just", "the", "rest" },
-                                                       (new File("src/test/resources/run/file"))
-                                                               .getAbsolutePath());
+        String[] args = new String[] {
+                "test_main_with_multiple_typed_params.bal", "1000", "1.0", "Hello Ballerina", "255", "true",
+                "{ \"name\": \"Maryam\" }", "<book>Harry Potter</book>", "{ \"name\": \"Em\" }", "just", "the", "rest"
+        };
+        String balFile = (new File("src" + File.separator + "test" + File.separator + "resources" + File.separator +
+                                           "run" + File.separator + "file")).getAbsolutePath();
+        String output = bMainInstance.runMainAndReadStdOut("run", args, balFile);
         Assert.assertEquals(output,
                             "integer: 1000, float: 1.0, string: Hello Ballerina, byte: 255, boolean: true, " +
                                     "JSON Name Field: Maryam, XML Element Name: book, Employee Name Field: Em, " +

--- a/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BasicJavaJarRunTestCase.java
+++ b/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/run/BasicJavaJarRunTestCase.java
@@ -1,0 +1,64 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.run;
+
+import org.ballerinalang.test.BaseTest;
+import org.ballerinalang.test.context.BMainInstance;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.LogLeecher;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+/**
+ * This class tests build a bal file and execute the jar via the java jar Command and the test the basic data
+ * binding functionality.
+ */
+public class BasicJavaJarRunTestCase extends BaseTest {
+
+    private static final int LOG_LEECHER_TIMEOUT = 10000;
+
+    @Test
+    public void testNoArg() throws BallerinaTestException {
+        BMainInstance ballerinaClient = new BMainInstance(balServer);
+        String serverResponse = "1";
+        String balFile = (new File("src" + File.separator + "test" + File.separator + "resources" + File.separator +
+                                           "run" + File.separator + "file" + File.separator +
+                                           "test_main_with_no_params.bal")).getAbsolutePath();
+        LogLeecher clientLeecher = new LogLeecher(serverResponse);
+        ballerinaClient.runMain(balFile, new LogLeecher[] { clientLeecher });
+        clientLeecher.waitForText(LOG_LEECHER_TIMEOUT);
+    }
+
+    @Test
+    public void testMultipleParam() throws BallerinaTestException {
+        BMainInstance ballerinaClient = new BMainInstance(balServer);
+        String serverResponse = "integer: 1000, float: 1.0, string: Hello Ballerina, byte: 255, boolean: true, JSON "
+                + "Name Field: Maryam, XML Element Name: book, Employee Name Field: Em, string rest args: just the "
+                + "rest";
+        String balFile = (new File("src" + File.separator + "test" + File.separator + "resources" + File.separator +
+                                           "run" + File.separator + "file" + File.separator +
+                                           "test_main_with_multiple_typed_params.bal")).getAbsolutePath();
+        LogLeecher clientLeecher = new LogLeecher(serverResponse);
+        ballerinaClient.runMain(balFile, null, new String[] {
+                "1000", "1.0", "Hello Ballerina", "255", "true", "{ " + "\"name\": \"Maryam\" }", "<book>Harry "
+                + "Potter</book>", "{ \"name\": \"Em\" }", "just", "the", "rest"
+        }, new LogLeecher[] { clientLeecher });
+        clientLeecher.waitForText(20000);
+    }
+}

